core/VectorState.hpp

```diff
@@ -10,19 +10,39 @@ namespace atlas
     class VectorState
     {
       public:
-        uint8_t getLMUL() const { return lmul_; }
+        inline uint8_t getLMUL() const { return lmul_; }
 
-        uint8_t getSEW() const { return sew_; }
+        inline uint8_t getSEW() const { return sew_; }
 
-        bool getVTA() const { return vta_; }
+        inline bool getVTA() const { return vta_; }
 
-        bool getVMA() const { return vma_; }
+        inline bool getVMA() const { return vma_; }
 
-        uint8_t getVL() const { return vl_; }
+        inline uint8_t getVL() const { return vl_; }
 
-        uint8_t getVSTART() const { return vstart_; }
+        inline uint8_t getVSTART() const { return vstart_; }
```

Comment:
There are no guarantees that adding `inline` will make a difference.  Most of the time the compiler will do it for you, so adding it to the header is superfluous.  No harm in adding it, but no advantage either.